src/tstester/base.ts

```diff
@@ -29,7 +37,7 @@ export abstract class TesterEvent {
 }
 
 // An error that fires when the after/within/tolerance in WaitTimingOptions is not met.
-class WaitTimeoutError extends Error {}
+export class WaitTimeoutError extends Error {}
```

Comment:
Do a name to this error

https://github.com/microsoft/jacdac-ts/blob/main/src/jdom/error.ts#L6

docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name

Reply:
do we need so many error types? Can we have a single error type with a enum flag attached to it?